.azure-pipelines/mgmt.yml

```diff
@@ -0,0 +1,27 @@
+pr:
+  branches:
+    include:
+    - dev
+
+pool:
+  vmImage: 'VS2017-Win2016'
+
+steps:
+  - task: CmdLine@1
+    inputs:
+      filename: C:\Program Files (x86)\Microsoft SDKs\Windows\v10.0A\bin\NETFX 4.6.2 Tools\sn.exe
+      arguments: -Vr *,31bf3856ad364e35
+  - task: MSBuild@1
+    inputs:
+      solution: build.proj
+  - task: MSBuild@1
+    inputs:
+      solution: build.proj
+      msbuildArguments: /t:RunTests /clp:ShowTimestamp /flp:Summary;Verbosity=minimal;LogFile=msbuild.sum /flp1:warningsonly;logfile=msbuild.wrn /flp2:errorsonly;logfile=msbuild.err /p:IgnorePathTokens="Microsoft.Rest.ClientRuntime.Tracing.Tests Microsoft.Azure.Services.AppAuthentication.IntegrationTests KeyVault\dataPlane LocationBasedServices Az.Auth.Tests BotService.Tests Search Batch.FileStaging.Tests Azure.Batch.IntegrationTests Azure.Batch.ProtocolTests Batch\Support\FileConventions AzureBatchFileConventions.IntegrationTests Batch\DataPlane\Azure.Batch.Unit.Tests Batch\DataPlane\Tools\ConfigureAwaitAnalyzer\ConfigureAwaitAnalyzer.Tests ContainerService\ContainerService.Tests Consumption\Consumption.Tests Billing\Billing.Tests ApplicationInsights\Management\ApplicationInsights.Tests Batch\Management\Management.Batch.Tests SecurityCenter\SecurityCenter.Tests AzureStack\Admin\FabricAdmin\Fabric.Admin.Tests IotHub\IotHub.Tests Monitor\Monitor.Tests Reservations\Reservations.Tests"
+  - task: PublishTestResults@2
+    condition: succeededOrFailed()
+    inputs:
+      testResultsFiles: '**/*.trx'
+      testRunTitle: 'Test Results for Windows builds'
```

Comment:
@shahabhijeet since we will split this into Windows (netframework+netcore) and Linux (netcore) builds I was wondering this could be a good title, let me know your thoughts

Reply:
My suggestion would be the following:

"Windows Test Results"
"Ubuntu Test Results"

I am not sure how much real estate we get in the UI, so that is the reason 'Windows/Ubuntu' should be in the begining.

Reply:
That seems reasonable. Is the plan to add the linux builds into this same yml and build pipeline?

Reply:
Yes, I will open a separate PR for the linux builds in the same yaml once this goes in